webui/react/src/pages/Admin/UserManagement.tsx

```diff
@@ -483,53 +483,59 @@ const UserManagement: React.FC<Props> = ({ onUserCreate }: Props) => {
     <>
       <Section className={css.usersTable}>
         <div className={css.actionBar} data-testid="actionRow">
-          <Row>
-            <Column>
-              <Row>
-                {/* input is uncontrolled */}
-                <Input
-                  allowClear
-                  data-testid="search"
-                  defaultValue={nameFilter}
-                  placeholder="Find user"
-                  prefix={<Icon color="cancel" decorative name="search" size="tiny" />}
-                  onChange={handleNameSearchApply}
-                />
-                <Select
-                  data-testid="roleSelect"
-                  options={roleOptions}
-                  searchable={false}
-                  value={roleFilter}
-                  width={120}
-                  onChange={handleRoleFilterApply}
-                />
-                <Select
-                  data-testid="statusSelect"
-                  options={statusOptions}
-                  searchable={false}
-                  value={statusFilter}
-                  width={170}
-                  onChange={handleStatusFilterApply}
-                />
-              </Row>
-            </Column>
-            <Column align="right">
-              <Row>
-                {selectedUserIds.length > 0 && (
-                  <Dropdown menu={actionDropdownMenu} onClick={handleActionDropdown}>
-                    <Button data-testid="actions">Actions</Button>
-                  </Dropdown>
-                )}
-                <Button
-                  aria-label={CREATE_USER_LABEL}
-                  data-testid="addUser"
-                  disabled={!info.userManagementEnabled || !canModifyUsers}
-                  onClick={CreateUserModal.open}>
-                  {CREATE_USER}
-                </Button>
-              </Row>
-            </Column>
-          </Row>
+          <div className={css.grid}>
+            <div className={css.gridInput}>
```

Comment:
are these wrapper divs necessary? my understanding of the grid is that it should automatically place these in the right area.

Reply:
i tried it and it seems to be fine. can you confirm?